NATIVE_SDK_VERSIONS.md

```diff
@@ -0,0 +1,12 @@
+| Unity SDK | iOS SDK | Android SDK |
+|----------|--------|------------|
+| 1.3.0 | 2.17.0 | 2.14.0 |
+| 1.2.0 | 2.14.1 | 2.11.0 |
+| 1.1.3 | 2.9.0 | 2.9.0 |
+| 1.1.2 | 2.9.0 | 2.9.0 |
+| 1.1.1 | 2.9.0 | 2.9.0 |
+| 1.1.0 | 2.9.0 | 2.9.0 |
+| 1.0.4 | 2.7.1 | 2.6.2 |
+| 1.0.3 | 2.7.1 | 2.6.1 |
+| 1.0.1 | 2.7.1 | 2.5.1 |
+| 1.0.0 | develop@e59ba8 | 2.5. |
```

Comment:
Missing the minor version here? Miscopied or wrong the Wiki? (It's probably 2.5.0)